csharp/ql/lib/semmle/code/csharp/dataflow/internal/SsaImpl.qll

```diff
@@ -310,7 +310,12 @@ private module CallGraph {
       c = any(DelegateCall dc | e = dc.getExpr()) and
       libraryDelegateCall = false
       or
-      c.getTarget().fromLibrary() and
+      exists(Callable target |
+        target = c.getTarget() and
+        not target.hasBody()
+      |
+        if target instanceof AddEventAccessor then not target.fromSource() else any()
```

Comment:
Why do we need to special case `AddEventAccessor`?

Reply:
Why don't we need to special case `RemoveEventAccessor`?

Reply:
Whoops, that's definitely a mistake. I think I will simply include all accessors, since the same holds for auto-generated getters/setters.